frappe/desk/page/setup_wizard/setup_wizard.py

```diff
@@ -84,7 +84,7 @@ def process_setup_stages(stages, user_input, is_background_task=False):
 	except Exception:
 		handle_setup_exception(user_input)
 		if not is_background_task:
-			return {"status": "fail", "fail": current_task.get("fail_msg")}
```

Comment:
This is used in UI for retries AFAIK :grimacing: 


What if we ship traceback in response and render it in UI somehow? See how error page does it (collapsed by default but available if required)

Reply:
use `frappe.get_tracback`

Reply:
also lets do `frappe.log_error` so it's persistent in DB.

Reply:
Ah okay, will do.

Reply:
Old:
![image](https://github.com/frappe/frappe/assets/10119037/439f4de9-b206-4f6f-a933-1f0c3fd6df3e)

<hr>
New:
![image](https://github.com/frappe/frappe/assets/10119037/81cbd9d0-2b9a-4453-9edf-394a3881ac2a)

Reply:
Retry still works fine after raising, will do the add to error log part

Reply:
(Console has full traceback)
![image](https://github.com/frappe/frappe/assets/10119037/77c6f311-a9d3-4243-a4d7-cd0f104da332)